packages/@aws-cdk/region-info/build-tools/generate-static-data.ts

```diff
@@ -52,6 +52,7 @@ async function main(): Promise<void> {
     'eu-west-2',
     'eu-west-3',
     'eu-north-1',
+    'me-south-1',
```

Comment:
Please leave those commented out - we don't yet have the CDK Metadata resource in those regions.

Reply:
Sure. Done.